lib/PulseAudio/Backend/Utilities.pm

```diff
@@ -14,18 +14,20 @@ our $_command_db;
 
 foreach my $name ( qw/card source source_output sink sink_input module client/ ) {
 	my $attr = $name . 's';
-	my $module = 'PulseAudio::' . ucfirst( lc $name );
-	
+	my $module = 'PulseAudio::' . join("", map(ucfirst, split(/_/, $name)));
```

Comment:
Existing logic turned sink_input into Sink_input instead of SinkInput so module was not loaded.